crates/burn-tensor/src/tests/quantization/ops/remainder.rs

```diff
@@ -62,11 +166,50 @@ mod tests {
         output
             .dequantize()
             .into_data()
-            .assert_approx_eq(&expected, 3);
+            .assert_approx_eq(&expected, 1);
     }
 
     #[test]
     fn should_have_no_remainder() {
+        // quantization errors introduces remainder values
+        let device = Default::default();
+        let lhs = TestTensor::<1>::from_data(
+            // [-1.4843, 1.135, -2.1563, 1.0862, 0.5034, 3.6587]
+            TensorData::quantized(
+                vec![-98i8, 17, -128, 15, -11, 127],
+                [6],
+                QuantizationStrategy::PerTensorAffineInt8(AffineQuantization::init(0.022804, -33)),
+            ),
+            &device,
+        );
+        let rhs = TestTensor::<1>::from_data(
+            // [1.4843, 1.135, 2.1563, 1.0862, 0.5034, 3.6587]
+            TensorData::quantized(
+                vec![-25i8, -49, 22, -52, -93, 127],
+                [6],
+                QuantizationStrategy::PerTensorAffineInt8(AffineQuantization::init(0.014363, -128)),
+            ),
+            &device,
+        );
```

Comment:
If you used symmetric quantization instead of affine, maybe the results would be much closer to no remainder? 🤔